src/openeo_gfmap/utils/catalogue.py

```diff
@@ -41,13 +43,15 @@ class UncoveredS1Exception(Exception):
 
 
 def _parse_cdse_products(response: dict):
-    """Parses the geometry of products from the CDSE catalogue."""
-    geoemetries = []
+    """Parses the geometry and timestamps of products from the CDSE catalogue."""
+    geometries = []
+    timestamps = []
     products = response["features"]
 
     for product in products:
-        geoemetries.append(shape(product["geometry"]))
-    return geoemetries
+        geometries.append(shape(product["geometry"]))
+        timestamps.append(pd.to_datetime(product["properties"]["startDate"]))
```

Comment:
Does every cdse product have a geometry and a startdate property?

To add some resilience I would add some try excepts here with warnings

Reply:
(i believe they should, but better safe than sorry?)

Reply:
I would think so, because startDate is the date of the acquisition, geometry is its footprint. But I just expanded what @GriffinBabe already implemented for the geometry, so I'm not sure. If these properties are not there, it's maybe not so bad to fail for now?

Reply:
preferable we would then log them and give a clear output error in the logging

Reply:
I added a check on the fields and logged in the case they don't exists

Although I never had an issue on that aspect when filtering catalogues, and I've been using this feature quite a lot already